scripts/aws_connect_db.sh

```diff
@@ -57,13 +57,6 @@ then
     exit 1
 fi
 
-which yq >/dev/null
-if [ $? -ne 0 ]
-then
-    echo "Please install yq - this is needed to interpret the required secret values."
-    exit 1
-fi
-
```

Comment:
Can we make this block check for `jq` instead?

Reply:
Added it back to check for `jq` instead.